pkgs/tools/misc/ncdu2/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, stdenv, fetchurl, fetchpatch, zig, ncurses }:
+
+stdenv.mkDerivation rec {
+  pname = "ncdu";
+  version = "2.0";
+
+  src = fetchurl {
+    url = "https://dev.yorhel.nl/download/${pname}-${version}.tar.gz";
+    sha256 = "sha256-Zs2mgEdnsukbeM/cqCX5/a9qCkxuQAYloBrVWVQYR8w=";
+  };
+
+  patches = [
+    ./c-import-order.patch # https://code.blicky.net/yorhel/ncdu/issues/183
+  ];
+
+  XDG_CACHE_HOME="Cache"; # FIXME This should be set in stdenv
+
+  nativeBuildInputs = [
+    zig
+  ];
+
+  buildInputs = [ ncurses ];
+
+  preInstall = ''
+    export PREFIX="$out"
+  '';
```

Comment:
This should be makeFlags.

Reply:
No, it needs to be an environment variable apparently.